aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -427,6 +429,14 @@ module supra_framework::pbo_delegation_pool {
         commission_pending_inactive: u64
     }
 
+    #[event]
+    struct UnlockScheduleUpdated has drop, store {
+        pool_address: address,
+        unlock_numerators: vector<u64>,
+        unlock_denominator: u64,
+        unlock_start_time: u64,
+        unlock_duration: u64
+    }
```

Comment:
Please add a new line here. I don't think the formatter will do this for you.

Reply:
done